.github/workflows/nightly-build.yaml

```diff
@@ -14,29 +14,18 @@
           name: "Set up node 12",
           uses: "actions/setup-node@v1",
           with: {
-            registry-url: "https://registry.npmjs.org",
+            registry-url: "https://registry.npmjs.com",
```

Comment:
Why do we switch from `.org` to `.com`?

Reply:
Our `.npmrc` specifies `.com` so I went for consistency 🤷

Reply:
That makes sense